test_schemas/en/test_list_collector.json

```diff
@@ -326,6 +326,43 @@
                             "id": "group-summary",
                             "title": "List Collector Summary",
                             "type": "SectionSummary"
+                        },
+                        {
+                            "type": "Question",
+                            "id": "test-number-block",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "test-currency",
+                                        "description": "",
+                                        "label": "Currency Test",
+                                        "mandatory": false,
+                                        "type": "Currency",
+                                        "currency": "GBP"
+                                    },
+                                    {
+                                        "id": "square-kilometres",
+                                        "description": "",
+                                        "label": "Kilometres Square",
+                                        "mandatory": false,
+                                        "type": "Unit",
+                                        "unit_length": "short",
+                                        "unit": "area-square-kilometer"
+                                    },
+                                    {
+                                        "id": "test-decimal",
+                                        "description": "",
+                                        "label": "Decimal test to 2 decimal places",
+                                        "mandatory": false,
+                                        "type": "Number",
+                                        "decimal_places": 2
+                                    }
+                                ],
+                                "description": "",
+                                "id": "test-number-range-question",
+                                "title": "Please enter test values (none mandatory)",
```

Comment:
Non-mandatory?

Reply:
I don't see a reason for it to be mandatory - is there something I'm missing?

Reply:
I mean the literal text😆

Reply:
:laughing: